app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,22 @@
+module SessionsHelper
+  # Logs in the given user.
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  # Returns the current logged-in user (if any).
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
||= là cái gì a nhỉ

Reply:
Toán tử này sẽ trả về giá trị của biến @current_user hoặc set giá trị cho biến này nếu nó nil (hoặc false) :))